ouroboros-consensus/src/unstable-mock-block/Ouroboros/Consensus/Mock/Ledger/Block.hs

```diff
@@ -170,7 +171,10 @@ data SimpleStdHeader c ext = SimpleStdHeader {
     , simpleBodySize :: SizeInBytes
     }
   deriving stock    (Generic, Show, Eq)
-  deriving anyclass (Serialise, NoThunks)
+  deriving anyclass (NoThunks)
+
+deriving anyclass instance KnownNat (Hash.SizeHash (SimpleHash c)) =>
```

Comment:
If this constraint appears in many more places I wonder if it would be wise to use an alias `KnownHashSize`

Reply:
Good idea.